db/seeds.rb

```diff
@@ -5,4 +5,15 @@
 # Examples:
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+  # Character.create(name: "Luke", movie: movies.first)
+  positions = Position.create([{ name: "Gerente" }, { name: "CEO" }, { name: "Assistente" }])
+  employees = Employee.create([{ name: "Jessie" }, { name: "Tony" }, { name: "Marie" }])
+  departments = Department.create([{ name: "Finanças", manager_id: 1 }, { name: "Design de Produto", manager_id: 2} ])
```

Comment:
vc pode usar os registros q vc ja criou:
```suggestion
  departments = Department.create([{ name: "Finanças", manager_id: employees[0] }, { name: "Design de Produto", manager_id: employees[1]} ])
```

Reply:
Yes, sur